public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -163,6 +166,7 @@ export class SharedPreferences extends PureComponent<Props, State> {
         queryHistory,
         navbar,
       });
+      this.setState({ isSubmitting: false });
```

Comment:
we should probably either wrap this in try/catch, or use promise methods to make sure `isSubmitting` is reset even if the call fails.